README.md

```diff
@@ -2,6 +2,28 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.1.2.
 
+## Prepping an Ubuntu VM:
+Prepping local ubuntu machine to build phylodiver-viz repo
+
+install npm, then angular using npm
+    https://angular.io/guide/setup-local
+
+1. `sudo apt-get install npm`
+2. `sudo npm install -g @angular/cli` 
+
+install an lts version of node 
+    https://github.com/nodesource/distributions/blob/master/README.md
+
+3. `curl -fsSL https://deb.nodesource.com/setup_lts.x > setup_lts.sh`
+4. `sudo -E bash setup_lts.sh`
+5. `sudo apt-get install -y nodejs`
+
+6. `ng version` #if this succeeds, angular cli is working
+
+install remaining project specific packages
+7. `npm install` 
```

Comment:
might want to mention you have to do this in the project top-level directory

Reply:
fixed